docs/stsynphot/appendixb_nonhst.rst

```diff
@@ -285,34 +285,32 @@ The Stromgren *uvby* throughputs are taken from
     >>> bp = stsyn.band('stromgren,y')  # doctest: +SKIP
 
 
-.. _stsynphot-nonhst-wfirst:
+.. _stsynphot-nonhst-roman:
 
-WFIRST
-------
+Roman
+-----
 
-Phase B estimates of the WFIRST integrated system throughputs have been taken from
-the `WFIRST Reference Information <https://wfirst.gsfc.nasa.gov/science/WFIRST_Reference_Information.html>`_ page at GSFC. For example:
+Phase C estimates of the Roman integrated system throughputs have been taken from
+the `Roman Reference Information <https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html>`_ page at GSFC. For example:
 
 >>> import stsynphot as stsyn
->>> bp = stsyn.band('wfirst,wfi,f062')  # doctest: +SKIP
```

Comment:
Just curious... Existing usage of `wfirst` will just provide and error now? Or do you secretly alias it to `roman` for backward compatibility?

Reply:
It should provide an error that the component is not found in the TMG/TMC. All of them have been renamed to Roman.